### ISSUE TITLE
1st Year study materials

### ISSUE THREAD

**Issue by astha2412** [arpit456jain/MMMUT-Notes-And-Study-Material-For-All-Year#1]
Will add 1st-year study materials for both semesters including(lecture notes, assignments, pdfs, syllabus)

**Comment by arpit456jain** [comment]
yes sure @astha2412 but its very hard to upload all this on Github
so you can upload it to your Google drive and add its shareable link in read me

**Comment by astha2412** [comment]
Yes I will add my drive's link
